index/org.go

```diff
@@ -288,3 +288,16 @@ func (index *OrgIndex) GetCA(name string) (string, error) {
 	// TODO - check existence
 	return index.Data.Body.CAs[name], nil
 }
+
+func (index *OrgIndex) GetCAs() (map[string]string, error) {
```

Comment:
Was thinking maybe to return and error if map is nil rather than empty.
On 2 Apr 2015 18:57, "Jon Bonazza" <notifications@github.com> wrote:

> In index/org.go
> <https://github.com/pki-io/core/pull/71#discussion_r27680847>:
>
> > @@ -288,3 +288,16 @@ func (index *OrgIndex) GetCA(name string) (string, error) {
> >  	// TODO - check existence
> >  	return index.Data.Body.CAs[name], nil
> >  }
> > +
> > +func (index *OrgIndex) GetCAs() (map[string]string, error) {
>
> Why does this have an error as tge second return type but never actually
> returns an error?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pki-io/core/pull/71/files#r27680847>.
>